### DISCUSSION TITLE
How we can update user data in latest version of next-auth

### DISCUSSION THREAD

**Discussion by kaelianbaudelet** [nextauthjs/next-auth#10242] [Help]
To update a session, you used to have to use the `update` method, which you `import { update } from "@/auth";`

Except that now I don't know how to do it. Could someone explain how to `update a session on the server side` in **latest beta next-auth (5.0.0-beta**)

**Comment by Bernardomennndes** [comment]
Actually, the `NextAuth` instance return a method called `unstable_update` which you could reassign the value to `update` for readability issues. Then, you just need to invoke the method on a RSC, using the "user server" tag if you want to pass to a child client component and make it interactable.

https://authjs.dev/reference/next-auth#unstable_update

**Comment by energetikk** [comment]
> To update a session, you used to have to use the `update` method, which you `import { update } from "@/auth";`
> 
> Except that now I don't know how to do it. Could someone explain how to `update a session on the server side` in **latest beta next-auth (5.0.0-beta**)

Hi, man! You found answer this problem?
